src/appmixer/clickup/bundle.json

```diff
@@ -1,7 +1,8 @@
 {
     "name": "appmixer.clickup",
-    "version": "1.0.1",
+    "version": "2.0.0",
     "changelog": {
+        "2.0.0": ["Updated CreateTask and FindTasks with dynamically loaded spaces and lists to select from. New components GetSpaces and GetLists."],
```

Comment:
@DavidDurman  please describe the breaking change and outline the steps for migration. also please move the message to the end of the list